### ISSUE TITLE
fatal: 'git@github.comsegmentio/analytics-ios.git' does not appear to be a git repository

### ISSUE THREAD

**Issue by jordeguevara** [segmentio/analytics-react-native#342]
I was running pod install and I got this error 

The package we are using is ```"@segment/analytics-react-native": "^1.3.2"```

Only reference to anything segment in code base aside from importing package is in package.json above.

Not sure where this is happening, any insight would be great. 

```

[!] Error installing Analytics
[!] /usr/bin/git clone https://github.com/segmentio/analytics-ios.git /var/folders/1q/1sz9_cwj60x7s9zcz0vynzqc0000gn/T/d20210721-16191-1megs0 --template= --single-branch --depth 1 --branch 4.1.5

Cloning into '/var/folders/1q/1sz9_cwj60x7s9zcz0vynzqc0000gn/T/d20210721-16191-1megs0'...
fatal: 'git@github.comsegmentio/analytics-ios.git' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

```

**Comment by migs647** [comment]
Looks like there is a colon missing between github.com and segmentio. CC @alanjcharles

**Comment by alanjcharles** [comment]
@jordeguevara can you try updating to `analytics-react-native 1.5.0`?  This has been fixed since 1.4.0 (#268). Please feel free to reach out to us at friends@support.com if the issue persists and we can go from there. Thanks!
